content/modules/r/pages/data-frames.adoc

```diff
@@ -0,0 +1,348 @@
+= data.frames
+
+{sp} +
+
+== Abstract
+
+Data.frames are one of the primary data structures used very frequently when working in R. Data.frames are tabular data structures, consisting of named columns of equal size, where each column has a single type.
```

Comment:
Data.frames are one of the most frequently used when working in R. 

Data.Frames organize data into a 2D table, using rows and columns. 
Each column contains values of one variable and each row contains one set of values for each column.